src/Command/OgAddFieldCommand.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Drupal\og\Command;
+
+use Drupal\Core\Entity\EntityTypeBundleInfo;
+use Drupal\Core\Entity\EntityTypeRepository;
+use Drupal\og\Og;
+use Drupal\og\OgFieldsPluginManager;
+use Drupal\og\Plugin\OgFields\AudienceField;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
+use Drupal\Console\Style\DrupalStyle;
+
+/**
+ * Class OgAddFieldCommand.
+ *
+ * @package Drupal\og
+ */
+class OgAddFieldCommand extends Command {
+
+  /**
+   * OG plugin manager.
+   *
+   * @var OgFieldsPluginManager
+   */
+  protected $ogFieldsPluginManager;
+
+  /**
+   * Bundle info service.
+   *
+   * @var EntityTypeBundleInfo
+   */
+  protected $bundleInfo;
+
+  /**
+   * Entity type repository service.
+   *
+   * @var EntityTypeRepository
+   */
+  protected $entityTypeRepository;
+
+  /**
+   * Constructor.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeBundleInfo $bundle_info
+   *   The bundle info service.
+   * @param \Drupal\Core\Entity\EntityTypeRepository $entity_type_repository
+   *   The type repository service.
+   * @param \Drupal\og\OgFieldsPluginManager $og_field_plugin_manager
+   *   The OG plugin manager.
+   */
+  public function __construct(EntityTypeBundleInfo $bundle_info, EntityTypeRepository $entity_type_repository, OgFieldsPluginManager $og_field_plugin_manager) {
```

Comment:
Here in the type hints it is really important to use the interfaces and not the services.